perception/sub8_perception/ml_classifiers/object_detection.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+import rospy
+from std_srvs.srv import SetBool, SetBoolResponse
+import subprocess
+import os
+import signal
+
+'''
+This is the main script for running Tensorflow inference. It a hacky
+solution to the problem that tensorflow leaves loaded graphs inside the GPU,
+even on 'session.close()'... And only clears it after processes has died.
+Hence, this script opens a ROS Enable service, that will open up a subprocess
+on enable, and close it on disable.
+'''
+
+# Exec names without the .py
+exec_names = ['dice_detection', 'path_localizer']
+
+# Store subprocesses in global
+alive_processes = {}
+
+
+class launcher(object):
+    def __init__(self, name):
+        '''
+        Object to store service threads and name of executable
+        param name: exectuable name assumbed to be in sub8_perception pkg
+                    which also enable a SetBoot ROS service under the path
+                    /{param name}/enable
+        '''
+        self.name = name
+        rospy.Service('/{}/enable'.format(name), SetBool, self.enable_callback)
```

Comment:
Change this to 
`` rospy.Service('/vision/{}/enable'.format(name), SetBool, self.enable_callback)``